lib/helpers/umzug-helper.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var args = require("yargs").argv;
+var path = require("path");
+var _    = require("lodash");
+
+module.exports = {
+  getStorageMethod: function () {
+    return args.storage || "sequelize";
```

Comment:
Yes, adding the two flags to the existing config file makes sense.  I'll work on that.

About your comment on checking the contents of the `sequelize-meta.json` file:  I could not understand how to use `readFile` defined in `test/support/helpers`.

Can you give me a hint on how I can do an `expect` using `readFile`?  I'm not familiar with `through2`.  Or should I just use `fs`?

Thank you for your feedback so far!